src/components/schedule-page/corrupted-schedule.component.tsx

```diff
@@ -0,0 +1,18 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import React from "react";
+import { EmptyMonthButtons } from "./empty-month-buttons";
+
+const MESSAGE =
+  "Nie można wyświetlić zapisanego grafiku.\n Spróbuj go ponownie wgrać lub, jeśli to możliwe, skopiować z poprzedniego miesiąca.";
```

Comment:
Moim zdaniem lepiej by brzmiało bez `jeśli to możliwe`, ponieważ tak na prawdę nie można będzie tego zrobić tylko jeśli przycisk `Wgraj plik` będzie wyszarzony
@pectom Co ty o tym sądzisz?

Reply:
To jest o kopiowaniu, bo ono nie jest zawsze możliwe, tylko jeśli poprzedni miesiąć jest valid. Z tym, że te komentarze jeszcze zmodyfikuję w kolejnym PR, gdy dodam powrót do poprzedniej wersji grafiku